atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -555,7 +558,12 @@ def do_worker_build(self, cluster_info):
         try:
             kwargs = self.get_worker_build_kwargs(self.release, cluster_info.platform,
                                                   self.koji_upload_dir, self.fs_task_id)
-            kwargs.update(override_kwargs)
+            if 'None' in override_kwargs:
+                kwargs.update(override_kwargs['None'])
```

Comment:
With `None` instead of `'None'`, yes. Can we also add a comment explaining that these parameters are overrides for all platforms?

Reply:
Fixed.